jsapp/js/universalTable/paginatedQueryUniversalTable.component.tsx

```diff
@@ -9,12 +9,25 @@ import type {UseQueryResult} from '@tanstack/react-query';
 import type {PaginatedResponse} from 'js/dataInterface';
 import type {UniversalTableColumn} from './universalTable.component';
 
+/**
+ * A way of passing custom options, e.g. a parameter that needs to be included
+ * in the API url.
+ */
+export interface PaginatedQueryHookOptions {
```

Comment:
Can we import `UseQueryOptions` from react-query instead?

Reply:
We have come up with an alternative approach that doesn't require passing options.